<?xml version="1.0" encoding="utf-8"?>
<tests>
  <compliant>
    <username>hardcoded</username>
    <environment>DB_USER=hardcoded</environment>
    <MasterUserPassword>!Ref 'MasterUserPassword'</MasterUserPassword>
    <db_password>null</db_password>
    <creds>DB_PASSWORD=set=it=here</creds>
    <passphrase>0</passphrase>
  </compliant>
  <noncompliant>
    <pwd>hardcoded1</pwd>
    <passwd>hardcoded2</passwd>
    <assigned>DB_PASSWORD=hardcoded3</assigned>
    <inlist>
      <item>DB_PASSWORD=hardcoded4</item>
    </inlist>
    <passphrase>hardcoded5</passphrase>
  </noncompliant>
</tests>